app/views/shared/_menu.html.erb

```diff
@@ -10,9 +10,9 @@
         </div>
         <li role="separator" class="mdc-deprecated-list-divider"></li>
       <% end %>
-      <%= drawer_menu_navigation_item('Inicio', root_path) %>
-      <%= drawer_menu_navigation_item('Materias', all_subjects_path) %>
-      <%= drawer_menu_navigation_item('Avance de Carrera', profile_path) %>
+      <%= drawer_menu_navigation_item('Avance de Carrera', root_path) %>
+      <%= drawer_menu_navigation_item('Todas las Materias', all_subjects_path) %>
+      <%= drawer_menu_navigation_item('Resumen por Grupo', profile_path) %>
```

Comment:
I noticed we're still showing `Avance de Carrera` as this [page's title](https://github.com/cedarcode/mi_carrera/blob/master/app/views/profiles/show.html.erb#L2), do we need to update that as well?

<img width="1313" alt="image" src="https://user-images.githubusercontent.com/62676004/226064848-d74ea6c8-8849-4ddf-8b53-d66378af2be5.png">